cmd/kubectl-gadget/bcck8s.go

```diff
@@ -192,6 +199,21 @@ func bccCmd(subCommand, bccScript string) func(*cobra.Command, []string) {
 			}
 		}
 
+		// biotop only works per node
+		if subCommand == "biotop" {
+			if containernameParam != "" || podnameParam != "" {
+				contextLogger.Fatalf("biotop doesn't support --containername or --podname")
+			}
+
+			if !allNamespaces {
+				contextLogger.Fatalf("biotop only works with --all-namespaces")
+			}
```

Comment:
I think this tools should also require the `--node` parameter. Otherwise in a multinode cluster the output of all nodes will be mixed.

Reply:
I'm standing on the opposite side. Large scale engineers are able to limit the command to one node, while others should use the tool as easy as possible. But it is up to you, i'm glad to add the extra restriction.

Anyway does the other commands have this mixing issue? Not all of them make the flag mandatory.

Reply:
I agree with Mauro, I think `--node` parameter should be mandatory unless we add a node column.

Reply:
I just tested it in a multinode cluster and I realised that even if we add the node column I don't think the output would have that much sense. It just confirmed for me that we need to ensure the user runs it on one single node at a time.

Reply:
Thanks, will update soon.

Reply:
Just to be clear. That would be very nice to support it in a multi node cluster but with the current logic that we have it's not possible. We have a component that mixes the output (lines of text) from the different nodes and shows it to the user. `biotop` doesn't work printing at line at the time but it prints something like a whole screen, so we don't have a way to mix those from different nodes at the moment.

Reply:
Thanks for the details. I have updated the PR.